src/Sylius/Bundle/UserBundle/Resources/config/services.xml

```diff
@@ -124,8 +124,8 @@
         </service>
         <service id="sylius.listener.password_updater" class="%sylius.listener.password_updater.class%">
             <argument type="service" id="sylius.user.password_updater" />
-            <tag name="kernel.event_listener" event="sylius.user.password_reset.success" method="genericEventUpdater" />
-            <tag name="kernel.event_listener" event="sylius.user.password_change.success" method="genericEventUpdater" />
+            <tag name="kernel.event_listener" event="sylius.user.password_reset.pre" method="genericEventUpdater" />
+            <tag name="kernel.event_listener" event="sylius.user.password_change.pre" method="genericEventUpdater" />
```

Comment:
I'd rename these events to: ``sylius.user.pre/post_password_change``. This is the convention we have for other events.